src/Symfony/Component/Messenger/RecordedMessageContainerInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Matthias Noback <matthiasnoback@gmail.com>
+ */
+interface RecordedMessageContainerInterface
```

Comment:
I am not sure about Container, it is just a simple collection. `RecordedMessagesInterface` should be enough IMHO.

Reply:
Hm.. I'm hesitant. Then we have two interfaces with super similar namses: `RecordedMessagesInterface` and `MessageRecorderInterface`. 
I wanted to use `RecordedMessageContainerInterface` so it is more explicit.

Reply:
Perfect